pandas/tests/series/test_dtypes.py

```diff
@@ -487,3 +488,241 @@ def test_reindex_astype_order_consistency(self):
         s1 = s.reindex(new_index).astype(temp_dtype).astype(new_dtype)
         s2 = s.astype(temp_dtype).reindex(new_index).astype(new_dtype)
         tm.assert_series_equal(s1, s2)
+
```

Comment:
can you split this test out to its own file: test_convert_dtypes

Reply:
done